CombinatorialGames/Surreal/Real.lean

```diff
@@ -388,12 +388,25 @@ theorem toSurreal_ratCast (q : ℚ) : toSurreal q = q := by
 @[simp, norm_cast] theorem toSurreal_one : toSurreal 1 = 1 := by simpa using toSurreal_natCast 1
 
 @[simp]
-theorem toSurreal_add (x y : ℝ) : toSurreal (x + y) = x + y := by
-  simpa using Surreal.mk_eq (toIGame_add_equiv x y)
+theorem toSurreal_neg (x : ℝ) : toSurreal (-x) = -toSurreal x :=
+  Surreal.mk_eq (toIGame_neg _).antisymmRel
 
 @[simp]
-theorem toSurreal_sub (x y : ℝ) : toSurreal (x - y) = x - y := by
-  simpa using Surreal.mk_eq (toIGame_sub_equiv x y)
+theorem toSurreal_add (x y : ℝ) : toSurreal (x + y) = x + y :=
+  Surreal.mk_eq (toIGame_add_equiv x y)
+
+@[simp]
+theorem toSurreal_sub (x y : ℝ) : toSurreal (x - y) = x - y :=
+  Surreal.mk_eq (toIGame_sub_equiv x y)
+
+@[simp]
+theorem toSurreal_max (x y : ℝ) : max x y = max (toSurreal x) (toSurreal y) := by
+  have := le_total x y
+  aesop
+
+@[simp, norm_cast]
+theorem toSurreal_abs (x : ℝ) : |x| = |toSurreal x| := by
+  simp [abs]
```

Comment:
Am I missing something here? The simp directions are wrong.

```suggestion
@[simp]
theorem toSurreal_max (x y : ℝ) : max (toSurreal x) (toSurreal y) = max x y := by
  have := le_total x y
  aesop

@[simp, norm_cast]
theorem toSurreal_abs (x : ℝ) : |toSurreal x| = |x| := by
  simp [abs]
```

(else, we get:)

Reply:
The lemma `toSurreal_max` actually reads
```
toSurreal (max x y) = max (toSurreal x) (toSurreal y)
```
I'm just removing the first `toSurreal` since it can be inferred. If it makes the lemma confusing to read I can add it back.